databricks_cli/runs/cli.py

```diff
@@ -39,21 +41,51 @@
               help='File containing JSON request to POST to /api/2.*/jobs/runs/submit.')
 @click.option('--json', default=None, type=JsonClickType(),
               help=JsonClickType.help('/api/2.*/jobs/runs/submit'))
+@click.option('--wait', is_flag=True,
+              help='If specified, the CLI will wait for the submitted run to complete ' +
+                   '(only available in API 2.1).')
 @api_version_option
 @debug_option
 @profile_option
 @eat_exceptions
 @provide_api_client
-def submit_cli(api_client, json_file, json, version):
+def submit_cli(api_client, json_file, json, wait, version):
     """
     Submits a one-time run.
 
     The specification for the request json can be found
     https://docs.databricks.com/api/latest/jobs.html#runs-submit
     """
     check_version(api_client, version)
-    json_cli_base(json_file, json, lambda json: RunsApi(
-        api_client).submit_run(json, version=version))
+    api_version = version or api_client.jobs_api_version
+    if api_version != '2.1' and wait:
+        click.echo(click.style('ERROR', fg='red') + ': the option --wait is only available ' +
+                   'in API 2.1', err=True)
+        return
+
+    if json_file:
+        with open(json_file, 'r') as f:
+            json = f.read()
+    submit_res = RunsApi(api_client).submit_run(json_loads(json), version=version)
+    click.echo(pretty_format(submit_res))
```

Comment:
Does it make sense to echo this?

Reply:
For this, I am just following the original behavior of the run submit CLI, which is this:
<img width="844" alt="Screen Shot 2022-06-15 at 11 37 35 AM" src="https://user-images.githubusercontent.com/66143562/173900656-8ef823f6-33ea-4648-bb3e-f51bf7e68859.png">